### PR TITLE
turbogit: update to version 1.3.3

### PR DESCRIPTION
**Pull request by b4nst**
#### Description

* update to version 1.3.3
* switch from release to archive

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 11.0.1 (20B50)
Xcode 12.3 12C33

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: devel/turbogit/Portfile [hunk 1]
```diff
@@ -13,12 +13,12 @@ maintainers         {b4nst @b4nst} openmaintainer
 description         Your git boat friend
 long_description    Cli tool built to help you deal with your day-to-day git work.
 
-github.tarball_from releases
-worksrcdir          .
+github.tarball_from archive
+worksrcdir          turbogit-${version}
```
Comment (reneeotten) [inline devel/turbogit/Portfile:17]: is setting `worksrcdir` required, I thought that the default values from the github PG should do the trick. Please see if you can remove this.

**Review by reneeotten — CHANGES_REQUESTED** [review]
you seem to be missing a dependency on `pkg-config`

File: devel/turbogit/Portfile (hunk 1 above)
Comment (b4nst) [reply devel/turbogit/Portfile]: Nice catch, thanks!

File: devel/turbogit/Portfile [hunk 2]
```diff
@@ -13,17 +13,18 @@ maintainers         {b4nst @b4nst} openmaintainer
...
+                    sha256  8f0facbd3cdc331efea8a11871e1556b572a1d4b8f89761836252c0b0f4b2d05 \
+                    size    1168134
 
 use_configure       no
 installs_libs       no
 
-depends_build       port:go
+depends_build       port:go \
+                    port:pkgconfig \
+                    port:libgit2
```
Comment (reneeotten) [inline devel/turbogit/Portfile:27]: it appears to me that `libgit2` is also required at runtime (and not only for building it):
```
% otool -L /opt/local/bin/tug
/opt/local/bin/tug:
	/opt/local/lib/libgit2.1.0.dylib (compatibility version 1.0.0, current version 1.0.1)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1281.100.1)
	/System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation (compatibility version 150.0.0, current version 1677.104.0)
	/System/Library/Frameworks/Security.framework/Versions/A/Security (compatibility version 1.0.0, current version 59306.140.5)

% port provides /opt/local/lib/libgit2.1.0.dylib
/opt/local/lib/libgit2.1.0.dylib is provided by: libgit2
```

so you should have that one listed as `depends_lib` instead.

Comment (b4nst) [reply devel/turbogit/Portfile:27]: Wow I'm tired! Thanks and yes obviously it's required at runtime, I'm using go binding... My bad this PR was a mess.

**Review by reneeotten — APPROVED** [review]

**Comment by reneeotten** [comment]
thanks @b4nst !

**Pull request merged**